build.yaml

```diff
@@ -817,6 +849,53 @@ steps:
     - default_ns
     - base_image    
     - test_batch2
+ - kind: runImage
+   name: create_ci_test_repo
+   image:
+     valueFrom: base_image.image
+   script: |
+     set -e
```

Comment:
Not having -x makes these scripts hard to debug, but there are insecure tokens in here.  What we've done elsewhere is turn on -x everywhere, and turn it off in the critical sections with an echo saying what's happening.  Something like:

```
set -ex

REPO_NAME="ci-test-{{token}}"

set +x
TOKEN=$(cat /secret/ci-secrets/user1)
echo creating $REPO_NAME...
curl -XPOST \
       -i \
       -sSL \
       https://api.github.com/orgs/hail-ci-test/repos \
       -H "Authorization: token ${TOKEN}" \
       -d "{ \"name\" : \"$REPO_NAME\" }"
set -x
...

Maybe not worth changing since this is working, but FYI.